### PR TITLE
feat(): Update make compare for no pret repo and add make for fork

### PR DESCRIPTION
**Pull request by KqesaR**
I make this PR to add jobs for symbols related issue #248 suggested by @Rangi42  and jobs in pokecrystal [https://github.com/pret/pokecrystal/blob/master/.github/workflows/main.yml#L58-L77](url)

I add also the update for `compare`jobs to disable `make compare` when is fork and a `make` when is fork

### CODE REVIEW

**Comment by Rangi42** [comment]
Thanks! But please remove the symbols creation; those aren't being added here (see #248).

**Comment by KqesaR** [comment]
@Rangi42 the PR is updated :) 

Sorry, I thought you were asking in the issue #248  to add the ability to add the symbols in a separate branch

File: .github/workflows/main.yml [hunk 1]
```diff
@@ -29,13 +29,23 @@ jobs:
...
       - name: Compare
+        if: ${{ github.repository_owner == 'pret' }}
         run: |
           make DEBUG=1 -j$(nproc) compare
           if ! git diff-index --quiet HEAD --; then
             echo 'Uncommitted changes detected:'
             git diff-index HEAD --
             return 1
           fi
+      - name: Make
```
Comment (Rangi42) [inline .github/workflows/main.yml:40]:
```suggestion

            - name: Make
```

File: .github/workflows/main.yml [hunk 2]
```diff
@@ -37,6 +38,16 @@ jobs:
             return 1
           fi
 
+            - name: Make
```
Comment (Rangi42) [inline .github/workflows/main.yml:41]:
```suggestion
      - name: Make
```

**Pull request merged**